apps/expo-go/android/app/proguard-rules.pro

```diff
@@ -147,9 +147,6 @@
 -keep class com.segment.analytics.** { *; }
 -keep class androidx.lifecycle.DefaultLifecycleObserver
 
-##### skia #####
--keep class com.shopify.reactnative.skia.** { *; }
```

Comment:
@Kudo will double check that this is correct

Reply:
tried release build expo-go and confirmed we can remove that safely. i guess it's because skia internally uses the @DoNotStrip annotations.